Solutions/BitSight/Analytic Rules/BitSight_Get_WFH_Data.yaml

```diff
@@ -0,0 +1,33 @@
+id: 54e77434-2184-435f-bc87-52b832ee451f
+name: BitSight Get WFH Data
+description: |
+  'Will generate an incident while new ip comes in networksession.'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: BitSight
+    dataTypes:
+      - ASimNetworkSession
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: GreaterThan
+triggerThreshold: 0
+query: |
+    ASimNetworkSession
+    | where EventSchema == "NetworkSession"
```

Comment:
@devikamehra 
We have specify that the user must needs to setup ASIM parser for already existing vendors in sentinel in the Readme file. We are not going to ship the ASIM parser for this as its not required the use case is as below.

Any user who is going to use this integration and they have different source of data in the sentinel other than the BitSight data and they want to use this Work From Home functionality then its their responsibility to create ASIM parser of NetworkSession schema and use this analytics rule which pass the other sources network session's src ip address to the playbook which would be executed using this analytics rule triggering. And then call the BitSight API with this mapped ip address and if data would be found related to this ip in the bitsight then it will store in the BitSight Work From Home custom table and same populated in the WFM dashboard of workbook.

Reply:
@jayeshprajapaticrest 
I think it is not how OOTB content works in Sentinel. Either we provide the ASIM parser out of the box and then give this query or we do not include this query at all.
Also, anything related to Network Session is already covered in Network Session Essential solution. We do not need to keep this in product specific solutions.
Please remove this use case for now.

Reply:
Plus, the query is not doing much here. It is creating Alert for all the Network Session information. This will be extremely noisy and irrelevant for any SOC Analyst.